### DISCUSSION TITLE
I can't get this to work using github.com/LazyVim/LazyVim

### DISCUSSION THREAD

**Discussion by BBennettChiba** [m4xshen/hardtime.nvim#105] [Q&A]
I don't know what to do. I've tried disabling all my personal plugins but every time I load nvim it doesn't respond to any of my keyboard inputs unless I hit : + enter. then it gives me the error
`Error executing  vim.on_key Lua callback: vim/_editor.lua:0: Error executing 'on_key' with ns_ids '3'
    Messages: Vim:E1174: String required for argument 1
stack traceback:
	[C]: in function 'error'
	vim/_editor.lua: in function <vim/_editor.lua:0>`

even after forcing responsiveness (by hitting `: + enter` the plugin doesn't work. Hardline show command works and it shows me some things but I get to notifications or messages.

I used the given lazyvim config in the readme. btw

I thought it might have something to do with keybindings so I disabled all of mine and I even disabled which-keys which is a default plugin with lazyvim/lazyvim but that did nothing.

**Comment by m4xshen** [comment]
Hi @BBennettChiba, what's your Neovim version? hardtime.nvim now requires nvim v0.10.0

**Reply by BBennettChiba** [reply]
damn. I thought it was up to date. apologies

**Reply by m4xshen** [reply]
> damn. I thought it was up to date. apologies

Never mind, that's my fault, I forgot to update the version checker.
